addons/mail/static/src/core/messaging_service.js

```diff
@@ -205,6 +205,32 @@ export class Messaging {
                         );
                     }
                     break;
+                case "discuss.channel/delete":
+                    {
+                        const thread = this.threadService.insert({
+                            id: notif.payload.id,
+                            model: "discuss.channel",
+                        });
+                        this.store.discuss.starred.messages =
+                            this.store.discuss.starred.messages.filter(
+                                (message) =>
+                                    message.resId !== thread.id && message.resModel !== thread.model
+                            );
+                        this.store.discuss.inbox.messages =
+                            this.store.discuss.inbox.messages.filter(
+                                (message) =>
+                                    message.resId !== thread.id && message.resModel !== thread.model
+                            );
+                        this.store.discuss.inbox.counter -= thread.message_needaction_counter;
```

Comment:
Note that if the variable in arrow function is `msg` instead of `message` it becomes a single line. Makes code easier to read